apps/catalogue/pages/[schema]/catalogue/[catalogue]/[resourceType]/[resource]/index.vue

```diff
@@ -272,6 +282,66 @@ function subpopulationMapper(subpopulation: any) {
   };
 }
 
+function variableMapper(variable: IVariable) {
+  const key = getKey(variable);
```

Comment:
offtopic more related to tailwind components: while I like some of the components in the catalogue, I am still a bit confused every time I see a mapper. I than consider that we already have a rich metadata system in emx2 and wonder if we could use that directly. So I would like to explore in the future if we can rebuild catalogue using the tailwind emx2 database backed components.